File: level 55/class/class.py
# N1
def small_enough(array, limit):
    return max(array)<=limit

# N2
def sum_two_smallest_numbers(numbers):
    return sum(sorted(numbers)[:2])

# N3
import math
def is_square(n):
    return n > -1 and math.sqrt(n) % 1 == 0;

# N4
CRITERIA = (str.islower, str.isupper, str.isdigit)

def password(s):
    return len(s)>7 and all( any(map(f,s)) for f in CRITERIA)

File: level 70/class/class.js
// N1
function sumTwoSmallestNumbers(numbers){  
  numbers = numbers.sort(function(a, b){return a - b; });
  return numbers[0] + numbers[1];
};


// N2
function isSquare(n) {
  return Math.sqrt(n) % 1 === 0;
}

// N3
function replace(s){
  return s.replace(/[aeoiu]/ig, '!')  
}